faculty_cli/cli.py

```diff
@@ -212,6 +225,16 @@ def _get_servers(project_id, name=None, status=None):
     return servers
 
 
+def _get_user_servers(user_id, status=None):
+    """List all servers owned by user."""
+    client = faculty.client("server")
+    endpoint = "/user/{}/instances".format(user_id)
+    servers = client._get(endpoint, ServerSchema(many=True))
```

Comment:
This should instead be added as a public method of the `ServerClient`.

Reply:
Yeah agreed, I've added the rest of the changes and will update when this is available via `ServerClient`